packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1196,7 +1198,7 @@ class _TabBarViewState extends State<TabBarView> {
 
     setState(() {
       _warpUnderwayCount -= 1;
-      _children = widget.children;
+      _children = originChildren;
```

Comment:
If `didUpdateWidget` gets called with `widget.children != oldWidget.children` and a wrap was underway (_warpUnderwayCount != 0) then we were relying on this line to update the children to the new widget.children. That no longer works now. (If there isn't a failing test for this, we should add one).